hedyorg/hedy
# Add 404 page to catch all unspecified requests

- Closes #1103 
- When adding 404 pages, I forgot to add a handler to catch any unspecified path. This is now fixed.

**How to test**:
- Go to http://localhost:5000/foo (see catch all page with message in English)
- Go to http://localhost:5000/foo?lang=nl (see catch all page with message in Dutch)
- Go to http://localhost:5000/foo?lang=es (see catch all page with message in Spanish)
